Tests/Remove-BBServerTag.Tests.ps1

```diff
@@ -0,0 +1,105 @@
+﻿# Copyright 2016 - 2018 WebMD Health Services
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#Requires -Version 5.1
+Set-StrictMode -Version 'Latest'
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-BitbucketServerAutomationTest.ps1' -Resolve)
+Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath '..\PSModules\GitAutomation') -Force
+
+$projectKey = 'GBBSTAG'
+$repo = $null
+$repoName = $null
+$repoRoot = $null
+$script:failed = $false

```

Comment:
You can remove the scope prefix since we're already at the script scope here. Also you'll want to add this to the `Init` function so it gets reset for every test.